rust_src/src/filelock.rs

```diff
@@ -0,0 +1,43 @@
+use remacs_macros::lisp_fn;
```

Comment:
Look at the other files and add a similar comment to the start of this one.

Reply:
Stole comment from `src/filelock.c`.

Reply:
Perfect 👍